client-core/src/lib.rs

```diff
@@ -0,0 +1,35 @@
+#![deny(missing_docs, unsafe_code, unstable_features)]
+//! # Crypto.com Chain Client
+//!
+//! This crate exposes following functionalities for interacting with Crypto.com Chain:
+//! - Wallet creation
+//! - Address generation
+//! - Balance tracking
```

Comment:
I suppose balance tracking is not implemented yet?

Reply:
No. Right now, only wallet creation and address generation are implemented. I'm still working on those features and also, a PR with all the features will become too big to review.